--- a/chrome/app/theme/chromium/BRANDING
+++ b/chrome/app/theme/chromium/BRANDING
@@ -1,10 +1,10 @@
-COMPANY_FULLNAME=The Helium Authors
-COMPANY_SHORTNAME=The Helium Authors
-PRODUCT_FULLNAME=Helium
-PRODUCT_SHORTNAME=Helium
-PRODUCT_INSTALLER_FULLNAME=Helium Installer
-PRODUCT_INSTALLER_SHORTNAME=Helium Installer
-COPYRIGHT=Copyright @LASTCHANGE_YEAR@ The Helium Authors. All rights reserved.
-MAC_BUNDLE_ID=net.imput.helium
+COMPANY_FULLNAME=Zenmium
+COMPANY_SHORTNAME=Zenmium
+PRODUCT_FULLNAME=Zenmium
+PRODUCT_SHORTNAME=Zenmium
+PRODUCT_INSTALLER_FULLNAME=Zenmium Installer
+PRODUCT_INSTALLER_SHORTNAME=Zenmium Installer
+COPYRIGHT=Copyright @LASTCHANGE_YEAR@ The Zenmium Authors. All rights reserved.
+MAC_BUNDLE_ID=com.zenmium.desktop
 MAC_CREATOR_CODE=Cr24
-MAC_TEAM_ID=S4Q33XPHB4
+MAC_TEAM_ID=
--- a/chrome/app/chrome_crash_reporter_client.cc
+++ b/chrome/app/chrome_crash_reporter_client.cc
@@ -130,7 +130,7 @@ bool ChromeCrashReporterClient::GetCrash
 void ChromeCrashReporterClient::GetProductInfo(ProductInfo* product_info) {
   CHECK(product_info);
-  product_info->product_name = "Helium";
+  product_info->product_name = "Zenmium";
   product_info->version = version_info::GetHeliumVersionNumber();
   product_info->channel =
       chrome::GetChannelName(chrome::WithExtendedStable(true));
